src/Main.js

```diff
@@ -0,0 +1,165 @@
+      </li>
+      <li className="nav-item">
+        <a className="nav-link" href="#">Link</a>
+      </li>
+    </ul>
+</nav>
+    );
+  }
+}
+
+// separation of selection box --> avatar, person's name, testimony
+
+/* Avatar */
+class Avatar extends React.Component {
+  render () {
+    return (
+      <img className="circle avatar" src={this.props.img} />
+      )
+  }
+}
+
+/* Testimony */
+class Testimony extends React.Component {
+  render () {
+    return (
+      <div className="d-flex p-3 align-items-center">
+        <div className="p-2">
+          <Avatar img={this.props.img} />
+          <p align="center"> {this.props.name}, {this.props.grade} </p>
+        </div>
+        <div className="p-2">
+          <p> {this.props.content} </p>
+        </div>
+      </div>
+      
+      )
+  } 
+}
+
+
+/* Dropdown options */
+class DropdownPrev extends React.Component {
+  render () {
+    return (
+  <div className="dropdown">
+  <button type="button" className="btn btn-primary dropdown-toggle" data-toggle="dropdown">
+    Subject
+  </button>
+  <div className="dropdown-menu">
+    <a className="dropdown-item" href="#">ACT</a>
+    <a className="dropdown-item" href="#">SAT</a>
+  </div>
+</div> )
+  }
+}
+
+// TODO: continue exploring how to loop over all the options posted 
+// so that we can use Dropdown generically
+
+// class DropdownOption extends React.Component {
+//   render() {
+//     return <a href={this.props.link}>{this.props.text}</a>;
+//   }
+// }
+
+/* More generalized dropdown option */
+// class Dropdown extends React.Component {
+//   renderDropdownOption(option, link) {
+//     return <NavbarOption text={option} link={link} />;
+//   }
+//   renderDropdownTitle() {
+//        <button type="button" className="btn btn-primary dropdown-toggle" data-toggle="dropdown">
+//         Subject
+//       </button>
+//   }
+//   render () {
+//     return (
+//       <div>
+//       this.renderDropdownTitle();
+//       var options = {this.props.options};
+//       for (var i = 0; i < options.length; i++) {
+//         this.renderDropdownOption(options[i]);
+//       }
+//       </div>
+//       )
+//   }
+// }
+
+/* Input box */
+class InputBox extends React.Component {
+  render () {
+    return (
+      <div className="container">
+        <div className="form-group">
+          <label for="usr">{this.props.title}</label>
+          <input type="text" className="form-control" id={this.props.id} />
+        </div>
+      </div> )
+  }
+}
+
+class SelectionBox extends React.Component {
+  render() {
+    return (
+        <div>
+          <InputBox id="usr" title="Username" />
+          <DropdownPrev />
+        </div>
+      )
+  }
+}
+
+
+
+/* Building the main page */
+class Main extends React.Component {
+  render() {
+    return (
+      <div>
+        <div>
+          <Navbar />
+        </div>
+        <div className="container-fluid row">
+        <div className="col-sm-5">
```

Comment:
Can we populate this with lengthier data so we can see what it looks like with a more realistic comment?